### ISSUE TITLE
[HL2DM][FeatureRequest] Make it FREE TO PLAY

### ISSUE THREAD

**Issue by motherofgod** [ValveSoftware/Source-1-Games#1697]
![12321321](https://f.cloud.github.com/assets/6983979/2492496/e48c7450-b232-11e3-8df5-fba6f80e98ed.jpg)
I'll show you some statistics from www.steamcharts.com.
This statistics clearly shows us that in 2015/16/17 online in this game will be less than 50 players.
This proves that the game must be made Free To Play, knowing policy Valve, you say that the game should payoff, but this is not a game like DoD, TF2 or CS: GO, is primarily a multiplayer mode, which has long ceased to be updated. 
I ask Valve to think about it or the game will come to desolation as Half-Life Deathmatch Source.
![1](https://f.cloud.github.com/assets/6983979/2490680/64e368b6-b1bb-11e3-84cf-026e764d1bc9.jpg)
![2](https://f.cloud.github.com/assets/6983979/2490681/64ed1cb2-b1bb-11e3-8589-115ce4ee8b71.jpg)

**Comment by Nubasik22** [comment]
Продолжаем трэд хотенья фритуплэя. Из 2014 в 2015. Поздравляю всех с новым годом и с началом нового учебного процесса.
А,......ну да....HALF LIFE 2 DM FREE TO PLAY HORAAAY

**Comment by mastercoms** [comment]
Guys, he already answered the question. There is no monetization option for them and the game is too old to be focused on. You can already play it for free if you have Source SDK.

**Comment by Yan-Tolstenov** [comment]
Half-Life 2 DM free to play is really cool!

**Comment by pumab** [comment]
Half-Life 2 DM free to play is really cool!

**Comment by Reedych** [comment]
Ni volas senpagan Half-Life 2 DM!

**Comment by TF2274** [comment]
Please make this game free to play! It's had a good run, probably made Valve a fair share of money, but there's no way it's worth being a paid game. If there were more players online then my argument would be different, but there's no way Valve is making any decent money off of the game.

**Comment by StrikerMan780** [comment]
I just wish they'd fix and maintain the damn thing.

**Comment by 1157** [comment]
Only 2 games get any support, TF2 and CS:GO (both sell stuff). Valve has forgot on how they got as big as they are,hl2,css,dod and the rest of the half-life series.

**Comment by CS-PRO1** [comment]
Agreed, HL2DM and DOD:S should go FTP if you want a better player base

**Comment by steve-Dundee** [comment]
When and if it ever goes F2P there will be at least 2 less servers to play on as I will either close mine or PW protect them to stop hackers and cheats getting back on with free accounts....

**Comment by Eyeroz** [comment]
yea it will be awsome to make this game f2p

**Comment by WonDerTeam1337** [comment]
Yeah make this game f2p please !!

**Comment by da11as911** [comment]
Please take into consideration the closed post mentioned above from all of the HL2DM community. Maybe a 20th anniversary gift to us all? Many of us have played since inception. Would be lovely to see, cheers!

**Comment by da11as911** [comment]
Update:
![image](https://github.com/user-attachments/assets/3cbedeed-8c06-44c7-ba7e-289c867e7038)

**Comment by WonDerTeam1337** [comment]
Yeah but they make it free there will be more player
